examples/end-to-end-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -74,10 +74,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    groupby_features_truncated = groupby_features_list &gt;&gt; nvt.ops.ListSlice(-SESSIONS_MAX_LENGTH, pad=False)
</pre><p>may be just remove <code>pad=False</code> from the ListSlice op. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
done